tutorials/01_introductory/usage.py

```diff
@@ -625,3 +625,95 @@ def my_plotter(ax, data1, data2, param_dict):
 # or generating a new set of figures.  In that case, use
 # :func:`~matplotlib.pyplot.show` to display the figure(s) and
 # to block execution until you have manually destroyed them.
+#
+# .. _performance:
+#
+# Performance
+# ===========
+#
+# Whether exploring data in interactive mode or programatically
+# saving lots of plots, rendering performance can be a painful
+# bottleneck in your pipeline. Matplotlib provides a couple
+# ways to greatly reduce rendering time at the cost of a slight
+# change (to a settable tolerance) in your plot's appearance.
+# The methods available to reduce rendering time depend on the
+# type of plot that is being created.
+#
+# Line segment simplification
+# ---------------------------
+#
+# For plots that have line segments (e.g. typical line plots,
+# outlines of polygons, etc.), rendering performance can be
+# controlled by the ``path.simplify`` and
+# ``path.simplify_threshold`` parameters in your
+# ``matplotlibrc`` file (see
+# :ref:`sphx_glr_tutorials_01_introductory_customizing.py` for
```

Comment:
This link might get changed by #8975, so need to be wary of whichever gets merged first...